### ISSUE TITLE
Pytorch Crush using backward.

### ISSUE THREAD

**Issue by IkeYang** [pytorch/pytorch#27742]
## 🐛 Bug

I am trying to use DCGAN with DCGAN TUTORIAL: https://github.com/pytorch/tutorials/blob/master/beginner_source/dcgan_faces_tutorial.py
But there occurs an error: Process finished with exit code -1073741819 (0xC0000005) During loss.backward() running.
## To Reproduce
My problem is  I can only train it on CPU. It raises an error when I try to train it on GPU.
Here is the error. And my cuda is available.
Process finished with exit code -1073741819 (0xC0000005) During 

the code is here https://github.com/IkeYang/AndrewNg_ML_practise/blob/master/GAN

## Environment
My environment is python3.6.9 windows10 torch1.2.0. cuda9.2
